opensrp-opd/src/main/java/org/smartregister/opd/OpdLibrary.java

```diff
@@ -33,12 +50,17 @@
     private OpdConfiguration opdConfiguration;
     private final Repository repository;
     private ECSyncHelper syncHelper;
+
     private UniqueIdRepository uniqueIdRepository;
+    private OpdCheckInRepository checkInRepository;
+    private OpdVisitRepository visitRepository;
+    private OpdDetailsRepository opdDetailsRepository;
+
     private Compressor compressor;
     private int applicationVersion;
     private int databaseVersion;
-    private OpdCheckInRepository checkInRepository;
-    private OpdVisitRepository visitRepository;
+
+    private SimpleDateFormat dateFormat = new SimpleDateFormat(OpdDbConstants.DATE_FORMAT, Locale.US);
```

Comment:
This line is repeated in at least 3 classes, kindly check if you can put it Utils or something

Reply:
I think the similarity might not require for the code to be shared due to the usage. What do you think. This link has been removed from the `OpdLibrary` but is present in other classes

Reply:
A wrapper would be nice so you can change the format or/Locale as you please